libs/site-base/package.json

```diff
@@ -25,7 +25,7 @@
     "test": "cross-env NODE_ENV=test jest --coverage",
     "test:watch": "cross-env NODE_ENV=test jest --watchAll",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
-    "extract": "formatjs extract \"src/**/!(*.d|*.test).ts*\" --out-file lang/en.json --id-interpolation-pattern [sha512:contenthash:base64:6]c",
```

Comment:
I was crushed when I saw all the ids for the translations changed after a yarn extract. 
Then I felt relieved to see that :)

Reply:
haha - yeah, we've wasted enough work already to lose any more!

Reply:
also... woops!!! How on earth did that sneak in there?  Good catch!